lib/src/bloc/movies_bloc.dart

```diff
@@ -19,8 +19,28 @@ class MoviesBloc extends IMoviesBloc {
   @override
   void fetchMovies() async {
     final movieList = await _moviesRepository.fetchAllMovies();
+    _sinkMovies(
+      movieList,
+    );
+  }
+
+  @override
+  void fetchMovieSearch(String movieKeywords) async {
+    if (movieKeywords != '') {
+      final movieList = await _moviesRepository.fetchMovieSearch(
+        movieKeywords,
+      );
+      _sinkMovies(
+        movieList,
+      );
+    } else {
+      this.fetchMovies();
+    }
+  }
+
+  void _sinkMovies(MoviesModel movies) {
```

Comment:
This method name can be _addFetchedMovies

Reply:
Done